src/ecobalyse_data/food/README.md

```diff
@@ -0,0 +1,63 @@
+# Utiliser la librairie python "Brightway2"
+
+La librairie python [brightway2](https://brightway.dev/) est un framework
+d'analyse du cycle de vie. Elle est très pratique pour pouvoir importer des
+données dans une base locale, faire des requêtes et calculs, et pour ensuite
+pouvoir exporter des données.
+
+Dans le répertoire [brightway/import_agb/](import_agb/) vous trouverez
+un script permettant d'importer la base de données
+[agribalyse](https://agribalyse.ademe.fr/), et dans le répertoire
+[brightway/export_agb/](export_agb/) le script utilisé pour exporter
+des données sous format json.
+
+Pour utiliser ces scripts, veuillez au préalable suivre la procédure
+d'installation et de création d'un environnement virtuel comme indiqué dans [le
+README](../README.md).
+
+Dans le répertoire
+[brightway/brightway_tutorial_notebooks/](brightway_tutorial_notebooks/)
+vous trouverez plusieurs exemples d'utilisation de la librairie brightway2 sous
+forme de [notebooks jupyter](https://jupyter.org/) pour explorer et expérimenter
+par vous-même.
+
+## Importer la base agribalyse v3.0 dans brightway
+
+Tout d'abord se placer dans le bon répertoire :
+
+    $ cd import_agb
+
+Attention : il est nécessaire d'avoir le fichier `agribalyse3_no_param.CSV` dans
+le répertoire. Ce fichier est un export de la base Agribalyse par le biais du
+logiciel (payant) SimaPro. Veuillez nous contacter si vous en avez besoin.
```

Comment:
Actually the file needed is `agribalyse3_no_param.CSV.zip` (note the `.zip` extension). If you just put the csv file, you get this error:

```
Creating core data migrations

Traceback (most recent call last):
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/import_agb/importing_databases.py", line 108, in <module>
    main()
  File "/home/niko/work/akei/ecobalyse/src/ecobalyse_data/food/import_agb/importing_databases.py", line 46, in main
    with ZipFile(agb_csv_zip_filepath) as zf:
  File "/home/niko/miniconda3/envs/ecobalyse-env/lib/python3.9/zipfile.py", line 1248, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'agribalyse3_no_param.CSV.zip'
```